src/UI/MasterUI.fl

```diff
@@ -87,6 +87,22 @@ decl {\#if !defined(PLUGINVERSION) && HAS_X11
 \#endif} {public local
 } 
 
+decl {\#if !defined(PLUGINVERSION) && HAS_X11
+\#include <FL/Fl_RGB_Image.H>
+\#endif} {private local
+} 
+
+decl {\#if !defined(PLUGINVERSION) && HAS_X11
+\#include <FL/FL.H>
+\#include "Enumerations.H"
```

Comment:
That should be #include <FL/Enumerations.H>, right?

Reply:
Enumerations.H is a file, not a class. In the header files and example code I read, it was included in this manner.

Reply:
I found a piece of example code (/test/sudoku.cxx) that includes it in this manner, so I'll make the change.